verilog/analysis/symbol_table_test.cc

```diff
@@ -1469,6 +1469,23 @@ TEST(BuildSymbolTableTest, ModuleInstanceNamedParameterAssignment) {
   }
 }
 
+TEST(BuildSymbolTableTest, KeywordAsIdentifierRegressionIssue917) {
```

Comment:
Maybe edit test name to describe the issue being tested?

Reply:
Done.